rte/main.go

```diff
@@ -225,20 +218,41 @@ func defaultHostName() string {
 	return val
 }
 
-func defaultTopologyManagerPolicy() string {
-	if val, ok := os.LookupEnv("TOPOLOGY_MANAGER_POLICY"); ok {
-		return val
+func setupTopologyManagerConfig(pArgs *ProgArgs, conf config.Config) error {
+	policyFrom := "args"
+	// do not overwrite with empty an existing value (e.g. from opts)
+	if pArgs.RTE.TopologyManagerPolicy == "" {
+		policyFrom = "env"
+		pArgs.RTE.TopologyManagerPolicy = os.Getenv("TOPOLOGY_MANAGER_POLICY")
```

Comment:
what if the environment variable doesn't align with the kubeletconfig settings? 
also may you tell please where is this env var is set?

Reply:
env is meant to override arguments. The priority is:
command line arguments overrides env vars overrides conf settings
(this is the standard UNIX way)
noone is setting these envs (anymore), it's for backward compat.